homeassistant/components/netatmo/services.yaml

```diff
@@ -46,6 +46,29 @@ set_preset_mode_with_end_datetime:
       selector:
         datetime:
 
+set_scheduled_room_temperature:
+  target:
+    entity:
+      integration: netatmo
+      domain: climate
+  fields:
+    zone_name:
```

Comment:
I checked netatmo's docs and it seems that they call this "custom temperatures" or "temperature sets" instead of "zones". Does it make sense to call this `temperature_set` instead of `zone_name`?

Reply:
Where did you find this "custom temperatures" terminology in the docs? In reality, Netatmo provides only temporary temperature adjustments that automatically revert to the predefined schedule after a specific duration. My objective is to effectively deactivate certain rooms based on occupancy. From what I understand, modifying the schedule's temperatures is the sole method to accomplish this.

Reply:
https://helpcenter.netatmo.com/hc/en-us/articles/360011029999-How-to-modify-the-weekly-schedule-how-to-apply-different-temperatures-room-by-room

There they call them "custom temperature sets"

Reply:
I finally understood what you were getting at. I renamed the method parameter. Pyatmo still uses the term "zones". But I guess it makes sense to expose them as temperature sets and stick to Netatmo's naming in Home Assistant.